config/initializers/new_framework_defaults_8_0.rb

```diff
@@ -0,0 +1,30 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
See comment on `config/application.rb`, is this file not redundant if we are already changing `config.load_defaults` there?